coderd/database/migrations/000094_add_resource_type_api_key.up.sql

```diff
@@ -0,0 +1,9 @@
+ALTER TYPE audit_action
+  ADD VALUE IF NOT EXISTS 'login';
+
+ALTER TYPE audit_action
+  ADD VALUE IF NOT EXISTS 'logout';
+
+ALTER TYPE resource_type
+  ADD VALUE IF NOT EXISTS 'api_key';
+
```

Comment:
If you run in a tx, this does not work. But we don't force migrations to be in a tx :thinking:. So this is ok :cry:

Reply:
Let me know if there's a better way! This is how we've been adding all audit enums in migrations.